_posts/2023-06-26-angular-standalone-components.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Angular standalone components"
+excerpt: "A simplified way to build Angular applications"
+tags: Web Angular standalone
+authors:
+- Ankit Pant
+header:
+  teaser: /assets/images/post/angular-standalone-components.png
+  teaser_alt: Angular standalone components
+category: Frontend
+---
+
+![](/assets/images/post/angular-standalone-components.png)
+
+# Introduction
```

Comment:
I miss saying on which Angular version the standalone component was introduced and what are the benefits of using standalone components.

Reply:
@tcorral I can add this to the introduction but the benefits are described in the performance section.

Reply:
@tcorral I have updated the documentation for the required information starting from line 18.